CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+### Added
+- `Kernel` base class and `MaternKernel` class
```

Comment:
should also add (here or in the net PR that adds more kernels) that the kernel type can now be specified instead if being hardcoded

Reply:
Suggestion: Make it two points as follows under Added:
- `Kernel` base class allowing to specify kernel types
- `MaternKernel` can be chosen for GP surrogates

Reply:
Changed it accordingly, please either resolve or propose different formulation.

Reply:
Since there is no comment/complaint regarding this, I interpret this as fine.